src/main/java/com/jcabi/github/FromProperties.java

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
@amihaiemil please add some tests for this class

Reply:
@llorllale Done; I'm not sure how to mock the IOException flow, though... I wrote 2 simple unit tests and the whole stuff is tested in the IT cases.